src/ty_module/early_binder.md

```diff
@@ -54,8 +54,8 @@ If they occur later, that is a compiler bug.
 
 [`FieldDef::ty`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/struct.FieldDef.html#method.ty
 [field_def_ty_src]: https://github.com/rust-lang/rust/blob/44d679b9021f03a79133021b94e6d23e9b55b3ab/compiler/rustc_middle/src/ty/mod.rs#L1421-L1426
-[`EarlyBinder`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/struct.EarlyBinder.html
-[`EarlyBinder::instantiate`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/struct.EarlyBinder.html#method.instantiate
+[`EarlyBinder`]: https://doc.rust-lang.org/nightly/nightly-rustc/stable_mir/ty/struct.EarlyBinder.html
```

Comment:
don't link to Stable MIR plz, its `EarlyBinder` != `rustc_middle::ty::EarlyBinder` as used in the actual compiler, its but a copy.

```suggestion
[`EarlyBinder`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/type.EarlyBinder.html
```

Reply:
Oh, good to know, thanks!